Services/Interfaces/web_interface/templates/components/config/profile_tab.html

```diff
@@ -216,6 +215,11 @@ <h5>
                 {% endif %}
             </div>
         </div>
+        <div class="card-footer">
+            {% if profile.read_only %}
+                <i class="fa fa-info"></i> This profile is ready only, press duplicate to create a copy to be able to edit it.
```

Comment:
by puttin this inside a <p> I think we will get a small padding that will make it look nicer